modules/ting/src/TingObject.php

```diff
@@ -0,0 +1,871 @@
+  }
+
+  /**
+   * Information about rights held in and over the resource.
+   *
+   * @return mixed
+   *   The rights.
+   */
+  public function getRights() {
+    return $this->rights;
+  }
+
+  /**
+   * List of materials in the same series as the this material.
+   *
+   * @return mixed
+   *   The materials.
+   */
+  public function getSeriesDescription() {
+    return $this->seriesDescription;
+  }
+
+  /**
+   * Title of the material from which this material stems.
+   *
+   * Eg. "Harry Potter and the philosopher's stone" is the source for
+   *  "Harry Potter und der Stein der Weisen"
+   *
+   * @return string
+   *   The title of the source.
+   */
+  public function getSource() {
+    return $this->source;
+  }
+
+  /**
+   * Spatial characteristics of the resource.
+   *
+   * Eg. "USA"
+   * Eg. "Italien"
+   *
+   * @return string
+   *   Spatial description of the material.
+   */
+  public function getSpatial() {
+    return $this->spatial;
+  }
+
+  /**
+   * Material spoken language.
+   *
+   * @return string
+   *   The language.
+   */
+  public function getSpoken() {
+    return $this->spoken;
+  }
+
+  /**
+   * Material subtitles.
+   *
+   * List of languages the material has subtitles in.
+   *
+   * Eg. ['Dansk', 'Tysk']
+   *
+   * @return string[]
+   *   List of languages.
+   */
+  public function getSubTitles() {
+    return $this->subTitles;
+  }
+
+  /**
+   * Titles of tracks in this material.
+   *
+   * @return string[]
+   *   Track titles.
+   */
+  public function getTracks() {
+    return $this->tracks;
+  }
+
+  /**
+   * Host publication URI.
+   *
+   * @return string
+   *   The uri formatted as a link.
+   */
+  public function getURI() {
+    return $this->uRI;
+  }
+
+  /**
+   * The revision of the material.
+   *
+   * Eg. "1. udgave, 1. oplag (2016)".
+   *
+   * @return string
+   *   The version.
+   */
+  public function getVersion() {
+    return $this->version;
+  }
+
+  /**
+   * Get relations for the material.
+   *
+   * Eg. "Berlingske tidende, 2005-08-20" (for a newspaper article).
+   *
+   * @return string[]
+   *   Relations.
+   */
+  public function isPartOf() {
+    return $this->isPartOf;
+  }
+
+  /**
+   * Gets the material type.
+   *
+   * Eg. "Bog"
+   * Eg. "Lydbog (cd)"
+   *
+   * @return string
+   *   The type.
+   */
+  public function getType() {
+    return $this->type;
+  }
+
+  /**
+   * Get the year the material was produced.
+   *
+   * Format YYYY.
+   *
+   * @return string
+   *   The year
+   */
+  public function getYear() {
+    return $this->year;
+  }
+
+  /**
+   * Returns the material classification.
+   *
+   * @return string|FALSE
+   *   The classification.
+   */
+  public function getClassification() {
+    return $this->classifications;
+  }
+
+  /**
+   * Returns a short for the material.
+   *
+   * @return string|FALSE
+   *   The abstract.
+   */
+  public function getAbstract() {
+    return $this->abstract;
+  }
+
+  /**
```

Comment:
Code sniffer:

> Missing short description in doc comment
> Missing parameter comment

This is true for the next _many_ functions!

Reply:
:+1: